.github/workflows/dockerize-profiles.yml

```
@@ -1,49 +1,49 @@
-        with:
-            images: appwrite/console-profiles
-            tags: |
-                type=ref,event=branch,prefix=branch-
-                type=ref,event=pr
-                type=sha,prefix=sha-
-                type=raw,value=gh-${{ github.run_id}}
-            flavor: |
-              latest=false
+        steps:
+            - name: Checkout the repo
+              uses: actions/checkout@v2
```

actionlint (1.7.8)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)